scripts/sv/callVariants.sh

```diff
@@ -21,12 +21,11 @@ echo "Assuming 2-bit reference: " "$TWOBIT_REFERENCE_LOCATION"
 
 cd "$GATK_DIR" 
 
-./gatk-launch CallVariantsFromAlignedContigsSpark \
-    --inputAlignments "$PROJECT_OUTPUT_DIR"/aligned_assemblies \
-    --inputAssemblies "$PROJECT_OUTPUT_DIR"/assembly_0 \
+./gatk-launch CallVariantsFromAlignedContigsSAMSpark \
```

Comment:
Unfortunately, yes... Not your fault. When I picked up the generalization to expand to calling insertions and deletions, both `CallVariantsFromAlignedContigsSAMSpark` and `CallVariantsFromAlignedContigsSpark` already exist. The `SAM` version, if I recall correctly, was used for 10X data, and the shorter one was used for the short reads.
The logic and parsing becomes separate somehow when I started inputing more functionalities into the parsing step used in `CallVariantsFromAlignedContigsSpark` (like breaking gapped alignment and dealing with edge case CIGARs). 

That said, it should not be a problem too difficult to fix. What I can do is to branch out from your branch, put in the fix in `AssemblyAlignmentParser` just for expected input, and PR into your branch (to relieve you from the pain of digging into the details of the file parsing as I have fresh memory of the details). What do you think?